profile/Overview.md

```diff
@@ -0,0 +1,66 @@
+
+# Confidential Containers
+We are interested in integrating existing [Trusted Execution Environments](https://en.wikipedia.org/wiki/Trusted_execution_environment) (TEE) 
+infrastructure support and technologies with the cloud native world. Our focus is to place a kubernetes pod into a TEE. 
+
+## Why?
+Security has long been a significant concern with data encryption at rest and in flight assumed to be a key part of any offering.
+Trusted Execution Environments look to address the data in use security concern.
+
+Cloud Computing adoption continues to accelerate whether it be Public, Private or increasingly common a Hybrid approach
+and with it the trust boundaries change. Consideration of Insider Threats needs to now consider the cloud provider,
+infrastructure provider, managed service provider.
+
+Certain Industries are heavily focused on compliance to standards, governments too are concerned both [collectively](https://www.un.org/counterterrorism/cybersecurity)
+and [individually](https://www.whitehouse.gov/briefing-room/presidential-actions/2021/05/12/executive-order-on-improving-the-nations-cybersecurity/).
+The standards expected to protect software solutions continue to evolve towards a concept of Confidential Computing
+
+Confidential Containers look to address the growing conern and needs of these three areas combining and growing in the future.
+
+## How?
+### Trusted Execution Environments
+![TEE Protects an application](../images/ApplicationTEEProtection.png)
+
+The TEE seeks to protect the Application and Data from outside threats, with the Application owner having complete control 
+of all communication across the TEE boundary. The application is considered a Virtual Machine and once supplied with the
+resources it requires, the TEE protects those resource (memory cpu) from the infrastructure and all communication across 
+the TEE boundary is under the control of the Application Owner.
+
+### Cloud Native Execution Environments
+![TEE protects an orchestrated pod](../images/CloudNativeTEEProtection.png)
+
```

Comment:
locking out the CSP while still being able to build cloud-native workloads. deprivileged orchestration etc